src/OrchardCore.Modules/OrchardCore.ContentFields/Startup.cs

```diff
@@ -105,6 +107,18 @@ public override void ConfigureServices(IServiceCollection services)
             services.AddScoped<IContentPartFieldDefinitionDisplayDriver, ContentPickerFieldSettingsDriver>();
             services.AddScoped<IContentFieldIndexHandler, ContentPickerFieldIndexHandler>();
             services.AddScoped<IContentPickerResultProvider, DefaultContentPickerResultProvider>();
+
+            services.AddScoped<IDataMigration, Migrations>();
+            services.AddScoped<IScopedIndexProvider, TextFieldIndexProvider>();
+            services.AddScoped<IScopedIndexProvider, BooleanFieldIndexProvider>();
+            services.AddScoped<IScopedIndexProvider, NumericFieldIndexProvider>();
+            services.AddScoped<IScopedIndexProvider, DateTimeFieldIndexProvider>();
+            services.AddScoped<IScopedIndexProvider, DateFieldIndexProvider>();
+            services.AddScoped<IScopedIndexProvider, ContentPickerFieldIndexProvider>();
+            services.AddScoped<IScopedIndexProvider, TimeFieldIndexProvider>();
+            services.AddScoped<IScopedIndexProvider, LinkFieldIndexProvider>();
+            services.AddScoped<IScopedIndexProvider, HtmlFieldIndexProvider>();
+            services.AddScoped<IScopedIndexProvider, YoutubeFieldIndexProvider>();
```

Comment:
@sebastienros Let me know if you want me to remove that one.

Reply:
Yes, don't do the YT one. We can create a video editor/display for the MEdia field instead. I need to find the PR where we said what we should do instead of this one.